### DISCUSSION TITLE
Does current timeseries branch saving the plots in mlflow for each model

### DISCUSSION THREAD

**Discussion by Chirag1168** [pycaret/pycaret#1730] [Q&A]
Like in classification and regression, does current timeseries branch saving the plots in mlflow for each model.

**Comment by ngupta23** [comment]
The plotting is natively built in the latest upcoming release of the time series module. You can see examples in the **[101 notebook here](https://nbviewer.org/github/pycaret/pycaret/blob/36e132eaf6b59835ec166bdd45fde115bb16ad91/time_series_101.ipynb)**.

Also `mlflow` logging is also enabled for time-series **[[Link](https://github.com/pycaret/pycaret/issues/1568#issuecomment-942222484)]**. 

Hope that answers your question.
